### PR TITLE
Update instructions for Cloud

### PR DESCRIPTION
**Pull request by hananiel**

### CODE REVIEW

File: CircuitBreaker/src/FortuneTeller/Fortune-Teller-UI/README.md [hunk 1]
```diff
@@ -53,16 +53,16 @@ Once you have the two applications communicating, you use of the Hystrix dashboa
...
 
 ### Setup Service Registry on CloudFoundry
 
 Using the service instance name of `myDiscoveryService`, complete the [common task](/CommonTasks.md#Spring-Cloud-Eureka-Server) of provisioning a Eureka server.
 
-### Setup Circuit Breaker Dashboard service on CloudFoundry
+### Setup Metrics Registrar Plugin 
 
-Using the service instance name of `myHystrixService`, complete the [common task](/CommonTasks.md#Hystrix-Dashboard) of provisioning a Hystrix dashboard.
+Hystrix dashboard on Spring Cloud Services has been deprecated, so we will have to use a metrics registrar to forward the Hystrix metrics to App Metrics.  Follow [these instructions](https://docs.pivotal.io/application-service/2-12/metric-registrar/using.html) to add Hystrix metrics emmitted by the app as custom metric source.
```
Comment (TimHess) [inline CircuitBreaker/src/FortuneTeller/Fortune-Teller-UI/README.md:65]:
```suggestion
Hystrix dashboard on Spring Cloud Services has been deprecated, so we will have to use a metrics registrar to forward the Hystrix metrics to App Metrics.  Follow [these instructions](https://docs.pivotal.io/application-service/2-12/metric-registrar/using.html) to add Hystrix metrics emitted by the app as custom metric source.
```

File: CircuitBreaker/src/FortuneTeller/Fortune-Teller-UI/README.md [hunk 2]
```diff
@@ -95,13 +95,13 @@ At this point the Fortune Teller UI is up and running and ready for displaying y
...
-### Using the Hystrix Dashboard - Cloud Foundry
+### Viewing the Hystrix Metrics - Cloud Foundry
 
-Once you have the two applications communicating, you can make use of the Hystrix dashboard by following the instructions below.
+Once you have the two applications communicating, you can view Hystrix metrics by following the instructions below.
 
 1. Open a browser or browser window and connect to the Pivotal Apps Manager.  You will have to use a link that is specific to your Cloud Foundry setup. (e.g. <https://apps.system.testcloud.com>)
-1. Follow [these instructions](https://docs.pivotal.io/spring-cloud-services/1-3/common/circuit-breaker/using-the-dashboard.html) to open the Hystrix dashboard service.
-1. Go back to the Fortune-Teller-UI application and obtain several fortunes.  Observe the values changing in the Hystrix dashboard.  Click the refresh button on the UI app quickly to see the dashboard update.
+1. Follow [these instructions](https://docs.vmware.com/en/App-Metrics-for-VMware-Tanzu/2.1/app-metrics/GUID-using.html#custom-metrics) to add Hystrix metrics emmitted by the app as custom metric source.
```
Comment (TimHess) [inline CircuitBreaker/src/FortuneTeller/Fortune-Teller-UI/README.md:103]:
```suggestion
1. Follow [these instructions](https://docs.vmware.com/en/App-Metrics-for-VMware-Tanzu/2.1/app-metrics/GUID-using.html#custom-metrics) to add Hystrix metrics emitted by the app as custom metric source.
```

File: CircuitBreaker/src/FortuneTeller/Fortune-Teller-UI/Startup.cs [hunk 3]
```diff
@@ -42,7 +42,8 @@ public void ConfigureServices(IServiceCollection services)
             services.AddControllersWithViews();
 
             // Add Hystrix metrics stream to enable monitoring
-            services.AddHystrixMetricsStream(Configuration);
+            //  services.AddHystrixMetricsStream(Configuration);
```
Comment (TimHess) [inline CircuitBreaker/src/FortuneTeller/Fortune-Teller-UI/Startup.cs:45]: If this line is still relevant for using the old dashboard, should we add a comment here that indicates when to use which?

File: CircuitBreaker/src/FortuneTeller/Fortune-Teller-UI/README.md (hunk 2 above)
Comment (hananiel) [reply CircuitBreaker/src/FortuneTeller/Fortune-Teller-UI/README.md:103]: I am about to update this to incorporate Dave's feedback on using the containerized dashboard as well.

File: CircuitBreaker/src/FortuneTeller/Fortune-Teller-UI/Startup.cs (hunk 3 above)
Comment (hananiel) [reply CircuitBreaker/src/FortuneTeller/Fortune-Teller-UI/Startup.cs]: Added comments to clarify usage

**Review by TimHess — APPROVED** [review]

**Pull request merged**